_posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Unity Space Simulator - Rotate Around VS Physics Based Approach
+date: 2019-11-04 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[This is part 2 of my Unity Space Series. You should first read part 1 if you haven't already][1]
+
+Recap: I'm playing around with Unity and I wanted to create a cube that would move through space until it got close enough to a planet. When that happened the cube would be attracted by the planet's gravity and start falling on it.
+
+In Unity there are 2 ways to do it. One using physics and the actual gravitational force of the planet against the gravitational force of the ship, another handling everything manually. We discussed how to implement the second one in the [previous article][1]. After way more testing I can confirm the approach that I took as being the best for my specific use case and wanted to weight the pros and cons of both methods.
```

Comment:
See now you call it a ship! Consistency!

Reply:
yes...